src/app/settings/settings-routing.module.ts

```diff
@@ -0,0 +1,17 @@
+import { NgModule } from '@angular/core';
```

Comment:
This module seems unnecessary since the settings component is never routed to so it isn't lazy loaded.

Reply:
Auto generated for pages I've removed it now.